host/libvirt_lxc_backend.go

```diff
@@ -464,6 +469,14 @@ func (l *LibvirtLXCBackend) Run(job *host.Job, runConfig *RunConfig) (err error)
 		}
 	}
 
+	for _, artifact := range job.TarArtifacts {
+		g.Log(grohl.Data{"at": "extract_tar", "uri": artifact.URI, "target_path": artifact.Attributes.TarTargetPath})
+		if err := fetchAndExtractTar(rootPath, artifact); err != nil {
```

Comment:
This was my initial thought but I put it in the host as then we can add artifact caching later. We could of course do the fetch in the host and extract in containerinit, but without caching we would then be fetching to disk unnecessarily.